src/lib/jsonParse/jsonAppendContextElementRequest.cpp

```diff
@@ -329,7 +329,9 @@ void jsonAcerRelease(ParseData* reqData)
 */
 std::string jsonAcerCheck(ParseData* reqData, ConnectionInfo* ciP)
 {
-  return reqData->acer.res.check(ciP, AppendContextElement, "", reqData->errorString, 0);
+  return reqData->acer.res.check(ciP->apiVersion,
+                                 ciP->uriParam[URI_PARAM_ATTRIBUTE_FORMAT] == "object" && ciP->outMimeType == JSON,
+                                 AppendContextElement, "", reqData->errorString, 0);
 }
```

Comment:
Two things here, if the function call is divided into lines, lets give each parameter a line each. Easier to read that way.
Secondly, the second parameter is a bit complex. A help-variable with a descriptive name would make it much easier to understand. E.g.:

    bool jsonObject = (ciP->uriParam[URI_PARAM_ATTRIBUTE_FORMAT] == "object") && (ciP->outMimeType == JSON);